test/parallel/test-abortcontroller.js

```diff
@@ -0,0 +1,29 @@
+// Flags: --no-warnings
+'use strict';
+
+require('../common');
+
+const { ok, CallTracker, strictEqual } = require('assert');
+
+const tracker = new CallTracker();
+process.on('exit', () => tracker.verify());
```

Comment:
Please keep on using `common.mustCall()` for now.

Reply:
I prefer to begin moving to the call tracker API here.

Reply:
That API might still change and we'll still have `mustCall()` around for a while. I would really rather use the other API for now.

Reply:
Changed to mustCall